qlty-cli/tests/cmd/smells/ignore_duplication_patterns.stderr

```diff
@@ -0,0 +1,4 @@
+     [0/3] [..]Analyzing all targets... [..]s
+     [1/3] [..]Checking structure of 4 files...  [..]s
+     [2/3] [..]Looking for duplication across 4 files...  [..]s
```

Comment:
I would delete this file, because the snapshot testing framework will not match against stderr if there is no file. This will make the test less brittle